### ISSUE TITLE
sanityCheck request "<<$"

### ISSUE THREAD

**Issue by pregAnon** [Free-Cities/Free-Cities#1033]
I don't understand it and I can't track down anyone else who does, so I'm extending my search here.

I'm seeking a simple check to find cases of `<<$` since they've cropped up enough times to me to be a nuisance. All signs point to no false positives either.

**Comment by prndev** [comment]
I am not quite sure what you are asking for. Are you in search for the regex pattern `'<<\$'` ?

**Comment by pregAnon** [comment]
Go to code `<<if $activeSlave` and then lose the if and space. You are left with the identifiable `<<$`. That is what I'm trying to catch.

**Comment by prndev** [comment]
Like so? 4172fd8

**Comment by pregAnon** [comment]
Ran a test. The "MissingSpaceBeforeVariable" is redundant with "VarSignMidWord", while "VarSignAtMacroStart" works like a charm and has already nailed something. Thank you.

**Comment by pregAnon** [comment]
Since you've been so helpful, I thought you might be able to figure out the possible why behind a new problem that has cropped up for me. It currently exclusively afflicts me, so don't worry too much about it.

"SlaveAttributeUsedOnce" is going off every instance of . before every variable. Swapping modded and unmodded sanityChecks doesn't change things. It seems to be something in the games code that is breaking the sanityCheck check. Any idea what could possible cause such a thing?

**Comment by pregAnon** [comment]
Update 3: Now it looks like it isn't the JS that was causing it. I'm back to square one again.

**Comment by prndev** [comment]
"SlaveAttributeUsedOnce" does not like my hackish work regarding HTML 5 color picker input either. I cannot see something that is totally off, though. Could you maybe push your intermediate work into a branch so I can have a look at everything in context?

PS: I just noticed that you also work on "Flexible Survival". Nice. 🙂

**Comment by pregAnon** [comment]
Would have to run a full comparison of the code to migrate the bits and pieces I want over without problem. Was planning to have it done nearly two weeks ago, but things have been very quiet around here, so with a lack of feedback, I've decided to handle mass RESS cleaning before starting to move code. Now the code is available, if you know where to look, but that's all I'm going to say.

Not really, I just wanted to poke about the code a little to see if I could spice up the pregnancy sytem a bit but couldn't get inform7 working right, so I gave up. That and the odd mix of how I much I dislike how furry it is vs how much I love the corruption aspect of it.
